src/v0/destinations/marketo_static_list/util.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable unicorn/consistent-destructuring */
```

Comment:
Why this?

Reply:
message.properties was causing some issues as was trying to modify the properties values ahead